app/models/room.js

```diff
@@ -67,7 +87,9 @@ RoomSchema.method('toJSON', function() {
         description: room.description,
         lastActive: room.lastActive,
         created: room.created,
-        owner: room.owner
+        owner: room.owner,
+        isPassworded: !!this.password
```

Comment:
Yeah, `isPassworded` sounds awkward. `hasPassword` seems like a better choice. `locked` might imply things beyond the room having a password.